src/components/Items/Question/Body.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-  <div class="overflow-y-scroll flex flex-col">
+  <div class="overflow-y-scroll flex flex-col py-4 shadow-inner">
     <!-- question text -->
-    <p class="m-2 sm:m-4 mx-4 md:mx-6 xl:mx-10 font-bold text-lg md:text-xl lg:text-2xl">
+    <p
+      class="m-2 sm:m-4 mx-4 md:mx-6 xl:mx-10 font-bold text-lg md:text-xl lg:text-2xl leading-tight"
+    >
       {{ questionText }}
     </p>
     <!-- option container -->
     <div class="flex mx-4 md:mx-6 xl:mx-10">
       <ul class="w-full">
-        <li class="list-none">
+        <li class="list-none gap-1 flex flex-col">
           <div
             v-for="(option, optionIndex) in options"
             :key="optionIndex"
-            class="p-2 sm:p-4 mb-2 text-lg md:text-xl lg:text-2xl rounded-md"
+            class="border p-2 pl-4 text-lg md:text-xl lg:text-2xl rounded-md mr-5"
```

Comment:
margin is best to not be added on only one side right?
`mx-5` instead of `mr-5`?